src/api/services/sendGridService.ts

```diff
@@ -0,0 +1,37 @@
+import sgMail from '@sendgrid/mail';
+
+import { SSP_SEND_GRID_API_KEY } from '../envars';
+import _templateIdMapping from '../templateIdMapping.json';
+import { EmailArgs, UID2Sender } from './emailTypes';
+
+type TemplateIdMapping = {
+  [key: string]: string;
+};
+const templateIdMapping = _templateIdMapping as TemplateIdMapping;
+
+sgMail.setApiKey(SSP_SEND_GRID_API_KEY);
```

Comment:
What happens if this is not set ? If this is mandatory, should we crash the app if this is not provided, or let it start silently, but add a warning?

Reply:
This is mandatory, I will add the warning to it